client/src/components/CodeEditor.jsx

```diff
@@ -34,12 +34,14 @@ import { viewSessionStore } from "../stores/viewSessionStore";
  * `onLanguageChange` is a callback function called when USER changes language
  */
 export const CodeEditor = observer(
-  ({ questionTitle, roomId, language, onLanguageChange, isGetNextQuestionLoading }) => {
+  ({ questionTitle, roomId, language, otherUsername, initialTemplate, onLanguageChange, isGetNextQuestionLoading }) => {
```

Comment:
questionTitle no longer used in file since u created a method for generating code template in the store, can delete?